src/closedWindows/closedWindowList/closedWindowList-controller.js

```diff
@@ -39,6 +45,10 @@
             this.$scope.$on('$destroy', () => {
                 window.windowService.removeClosedWindowListener(listener);
             });
+
+            this.$scope.$on('windowClosed', () => {
+                this.overriddenIcon = this.$scope.icon + '_active';
+            });
```

Comment:
I'm not sure if I see what you mean -- I see only one listener for `windowClosed`..